system/utils.go

```diff
@@ -55,6 +55,43 @@ func (u User) AssignGroup(
 	return finalMap, nil
 }
 
+//go:embed edit_queries/user_delete_group.graphql
+var userDeleteGroupQueryString string
```

Comment:
Could also be renamed more appropriately

Reply:
👍🏻 I have standardized all the names (assign / unassign) in utils.